src/librustc_trans/builder.rs

```diff
@@ -1105,12 +1105,6 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
         }
     }
 
-    pub fn set_personality_fn(&self, personality: ValueRef) {
```

Comment:
Could we keep this but change it to call `llvm::LLVMSetPersonalityFn(self.llfn(), llpersonality);` instead of inlining it? That would keep the unsafe more local to `builder.rs` which is a win, I believe.

Granted this does save us one builder to function transition, but I don't think (hope) that's an expensive operation.